php-cs-fixer

```diff
@@ -11,28 +13,33 @@
  * with this source code in the file LICENSE.
  */
 
+use Composer\XdebugHandler\XdebugHandler;
+use PhpCsFixer\Console\Application;
+
 error_reporting(E_ALL & ~E_DEPRECATED & ~E_USER_DEPRECATED);
 
-set_error_handler(static function ($severity, $message, $file, $line) {
-    if ($severity & error_reporting()) {
+set_error_handler(static function (int $severity, string $message, string $file, int $line): bool {
+    if (0 !== ($severity & error_reporting())) {
         throw new ErrorException($message, 0, $severity, $file, $line);
     }
+
+    return true;
 });
 
 // check environment requirements
-(function () {
-    if (\PHP_VERSION_ID === 80000) {
+(static function (): void {
+    if (\PHP_VERSION_ID === (int) '80000') {
```

Comment:
:(

Reply:
option 1: don't put workaround in code to please SCA tool. have proper solution or improvement on SCA, avoid workaround. even if SCA tool is Fixer itself

option 2: run check for entry-file with dedicated config file or under lowest allowed PHP version (alignment needed, or keep status quo of 5.6 - ref https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8229#discussion_r1783512152) [[ I'm not pushing to keep 5.6 as minimum version file is compatible with ]]

option 3: do the ugly workaround but add in-code comment whats going on with TODO how it should look alike when workaround no longer needed. otherwise knowledge of workaround background is lost

Reply:
what issue having `if (\PHP_VERSION_ID === 80000) {` was reported?

Reply:
`80000` is formatted (by fixer) as `80_000` which is non PHP 7.0 compatible

Reply:
can we at least apply that part?
> option 3: do the ugly workaround but add in-code comment whats going on with TODO how it should look alike when workaround no longer needed. otherwise knowledge of workaround background is lost

Reply:
My pleasure, I completely missed this TODO. Now everything should be addressed.